recipes/func-adl-xAOD/recipe.yaml

```diff
@@ -0,0 +1,53 @@
+schema_version: 1
+
+context:
+  name: "func-adl-xAOD"
+  version: "2.2.4"
+  python_min: "3.9"
```

Comment:
Intentionally setting this even though `3.9` is the default, as this library belongs to an ecosystem of tools that track the versions of Python used in the experimental high energy physics landscape, which moves slower to adopting new Pythons and dropping older versions.